src/components/color-picker/helpers/checkboard.js

```diff
@@ -22,14 +22,14 @@ export function render(c1, c2, size, serverCanvas) {
   return canvas.toDataURL();
 }
 
-export function get(c1, c2, size, serverCanvas) {
-  const key = `${c1}-${c2}-${size}${serverCanvas ? '-server' : ''}`;
+export function get(c1, c2, size, ServerCanvas) {
+  const key = `${c1}-${c2}-${size}${ServerCanvas ? '-server' : ''}`;
 
   if (checkBoardCache[key]) {
     return checkBoardCache[key];
   }
 
-  const checkBoard = render(c1, c2, size, serverCanvas);
+  const checkBoard = render(c1, c2, size, ServerCanvas);
```

Comment:
Why is this file not called 'checkBoard.js' but 'checkboard.js' ? I think it should use camelcase.

Please take care that when changing only the case it is better to use `git mv checkboard.js checkBoard.js` in order to avoid trouble with non case sensitive FS.

Reply:
fixed